features/ui/footer/footer.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import { Routes } from "@config/routes";
+import { color, breakpoint, textFont, space } from "@styles/theme";
+import { version } from "config/version";
+import { FooterItemLink } from "./footer-item-link";
+
+const footerItems = [
+  { text: "Docs", href: Routes.docs },
+  { text: "API", href: Routes.api },
+  { text: "Help", href: Routes.help },
+  { text: "Community", href: Routes.community },
+];
+
+const Container = styled.footer`
+  display: grid;
+  grid-template-areas:
+    "nav"
+    "logo"
+    "version";
+  padding: ${space(6)};
+  height: calc(
```

Comment:
I think you can use `box-sizing: border-box` instead of calculating the height. Made the same mistake while building this app 😄